HACKING.md

```diff
@@ -0,0 +1,55 @@
+# Pod Checkpointer Operator Hacking
+
+## Local development
+
+It's possible (and useful) to develop the operator locally targeting a remote cluster.
+
+### Prerequisites
+
+* An OpenShift or Kubernetes cluster with at least a master, infra, and compute
+node. An admin-scoped * `KUBECONFIG` for the cluster. The [operator-sdk](https://github.com/operator-framework/operator-sdk).
+
+### Building
+
+To build the operator during development, use the standard Go toolchain:
+
+```
+$ go build ./...
```

Comment:
We have a makefile.  Maybe change this to `make build`.

Reply:
fixed in 9495d2b